### PR TITLE
SDCORE-761: PLMN update does not work in SMF

### PR DESCRIPTION
**Pull request by UpadhyayaAnkur**

### CODE REVIEW

File: factory/config.go [hunk 1]
```diff
@@ -482,6 +482,19 @@ func compareUPNodesConfigs(u1, u2 map[string]UPNode) (match bool, add, mod, del
 
 }
 
+func compareNetworkSliceInstance(s1, s2 SnssaiInfoItem) (match bool) {
```
Comment (thakurajayL) [inline factory/config.go:485]: Can you add intel copyright in this file ?

Comment (UpadhyayaAnkur) [reply factory/config.go:485]: Done

File: factory/config.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
 // Copyright 2019 free5GC.org
-//
+// SPDX-FileCopyrightText: 2022-present Intel Corporation
```
Comment (thakurajayL) [inline factory/config.go:3]: New copyright at top

Comment (UpadhyayaAnkur) [reply factory/config.go]: Done

**Comment by UpadhyayaAnkur** [comment]
retest this please

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**